domain/appeal.go

```diff
@@ -120,6 +120,7 @@ type ApprovalAction struct {
 type ListAppealsFilter struct {
 	CreatedBy                 string    `mapstructure:"created_by" validate:"omitempty,required"`
 	AccountID                 string    `mapstructure:"account_id" validate:"omitempty,required"`
+	AccountType               string    `mapstructure:"account_type" validate:"omitempty,required"`
```

Comment:
@singhvikash11 Does account_type reflect that it is active or inactive? If yes, should it be called `account_status`? And can we also think about what possible values it can have?

Reply:
`account_type` was already there in appeal model, it is for specifying the type of `account_id` e.g.: `user` or `serviceAccount`